src/pages/search/project_search/ProjectSearch.tsx

```diff
@@ -24,14 +25,22 @@ export const ProjectSearch = () => {
   if (!projectSearchQueryParams.get(SearchParam.Results)) {
     projectSearchQueryParams.set(SearchParam.Results, '10');
   }
+  if (!projectSearchQueryParams.get(SearchParam.Page)) {
+    projectSearchQueryParams.set(SearchParam.Page, '1');
+  }
 
-  const queryParams = projectSearchQueryParams.toString();
+  const rowsPerPage = Number(projectSearchQueryParams.get(SearchParam.Results));
+  const page = Number(projectSearchQueryParams.get(SearchParam.Page));
+  const titleQuery = projectSearchQueryParams.get(SearchParam.Query);
 
-  const [projectsSearch, isLoadingProjectsSearch] = useFetch<SearchResponse<CristinProject>>({
-    url: queryParams ? `${CristinApiPath.Project}?${queryParams}` : '',
-    errorMessage: t('feedback.error.project_search'),
+  const projectsQuery = useQuery({
+    queryKey: ['projects', rowsPerPage, page, titleQuery],
+    queryFn: () => searchForProjects(rowsPerPage, page, { query: titleQuery ?? undefined }),
+    onError: () => dispatch(setNotification({ message: t('feedback.error.search'), variant: 'error' })),
```

Comment:
`onError` er deprecated (ser det om du hovrer over, så bruk meta i stedet:
```suggestion
    meta: { errorMessage: t('feedback.error.search') },
```